jupyter-img/druidapi/druidapi/sql.py

```diff
@@ -809,6 +809,19 @@ def rows(self):
                     page+=1
         return self._rows
```

Comment:
A note for the line at the start re: the version of Druid required for this notebook (it says 28).

Also, that the old notebook for 28 should be retained with a suffix of "-28" as per the lookups one.

Reply:
Let's talk about this... I don't think we want this growing list of versioned notebooks.
Regarding it not working for you, it seems like you were using the old jupyter image. How did you start the docker compose?

Reply:
Applied new versioning strategy including link in README